docs/Project.toml

```diff
@@ -5,5 +5,5 @@ Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
 
 [compat]
 Documenter = "1.10.0"
-ITensorNetworks = "0.14.0"
+ITensorNetworks = "0.14.1"
```

Comment:
Just a note that you only need to update the `Project.toml` compat entries in subdirectories for breaking releases.